script.js

```diff
@@ -116,7 +124,36 @@ const app = {
     app.addEventToTasks();
   },
 
+  getSelectedValue: function() {
+    let arrayTask = Object.values(this.tasks);
+    var selectedValue = document.getElementById("select_id").value;
+    var arraySorted = [];
+    if (selectedValue == "Title-Asc") {
+      arraySorted = arrayTask.sort(this.compare);
+    }
+    if (selectedValue == "Title-Desc") {
+      arraySorted = arrayTask.sort(this.compareDesc);
+    }
+    if (selectedValue == "Due-Date-Asc") {
+      arraySorted = arrayTask.sort(this.compareDateAsc);
+    }
+    if (selectedValue == "Due-Date-Desc") {
+      arraySorted = arrayTask.sort(this.compareDateDesc);
+    }
+    if (selectedValue == "Creation-Date-Asc") {
+      arraySorted = arrayTask.sort(this.compareCreateDateAsc);
+    }
+    if (selectedValue == "Creation-Date-Desc") {
+      arraySorted = arrayTask.sort(this.compareCreateDateDesc);
+    }
+    if (selectedValue == "Order by") {
+      arraySorted = arrayTask.sort(this.compareIdDesc);
+    }
```

Comment:
I think could be better they using `case` like this:
```js
switch (selectedValue) {
  case "Title-Asc":
    arraySorted = arrayTask.sort(this.compare);
    break;
  ...
  case "Order by":
    arraySorted = arrayTask.sort(this.compareIdDesc);
    break;
}
```

Reply:
Could you explain why do you think that?